### PR TITLE
Pia 3345 navigation buttons customization

### PR DESCRIPTION
**Pull request by vizaknaidavid**

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Custom views/GiniBarButton.swift [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+// GiniBarButton.swift
+//  
+//
+//  Created by David Vizaknai on 15.03.2023.
+//
+
+import UIKit
+
+public enum BarButtonType {
+    case cancel, help, back(title: String), done
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Custom views/GiniBarButton.swift:11]: [Suggestion] Could you please move each case per line 😇?

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/GiniConfiguration.swift [hunk 2]
```diff
@@ -145,6 +145,28 @@ import GiniBankAPILibrary
...
+
+    /**
+     Sets custom icon for the every Help button on the top navigation bar
+     */
+    public var giniNavigationBarButtonHelpButtonIcon: UIImage?
+
+    /**
+     Sets custom icon for the every Back button on the top navigation bar
+     */
+    public var giniNavigationBarButtonBackButtonIcon: UIImage?
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/GiniConfiguration.swift:164]: [Clarification] will it work if we set in the SDK default image.asset with an empty image and if the user decides to override it then the image will be added to the nav button?🤔

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Help/DigitalInvoiceHelpViewController.swift [hunk 3]
```diff
@@ -58,6 +58,10 @@ final class DigitalInvoiceHelpViewController: UIViewController {
         scrollView.addSubview(contentView)
         contentView.addSubview(stackView)
 
+        let backButton = GiniBarButton(ofType: .back(title: "Digitalinvoice+"))
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Help/DigitalInvoiceHelpViewController.swift:61]: [Clarification] Why do we have the titles like "Digitalinvoice+"?

**Review by zladzeyka — COMMENTED** [review]
Looks great. Just a few clarifying questions from my side 😇

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Help/DigitalInvoiceHelpViewController.swift (hunk 3 above)
Comment (vizaknaidavid) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Help/DigitalInvoiceHelpViewController.swift]: It was jist for debugging

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/GiniConfiguration.swift (hunk 2 above)
Comment (vizaknaidavid) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/GiniConfiguration.swift]: It worked with this approach. :D Thx for the idea 🙏

**Review by zladzeyka — APPROVED** [review]
Looks good 👍

File: BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppCoordinator.swift [hunk 4]
```diff
@@ -59,6 +59,11 @@ final class AppCoordinator: Coordinator {
 //        configuration.digitalInvoiceOnboardingNavigationBarBottomAdapter = CustomDigitalInvoiceOnboardingBottomNavigationBarAdapter()
 //        configuration.digitalInvoiceNavigationBarBottomAdapter = CustomDigitalInvoiceBottomNavigationBarAdapter()
 
+//        configuration.giniNavigationBarBackButtonIcon = UIImage(named: "minus_icon")
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppCoordinator.swift:62]: @vizaknaidavid Could you please remove it, you don't use those lines anymore ;)

**Review by zladzeyka — APPROVED** [review]
Great job 👍 only one little artifact left.

**Pull request merged**